packages/visualizations/addon/components/cell-renderers/metric.js

```diff
@@ -52,6 +54,6 @@ export default Ember.Component.extend({
       return numeral(metricValue).format(format);
     }
 
-    return metricValue;
+    return type === 'metric' ? smartFormatNumber([metricValue]) : metricValue;
```

Comment:
Do we need the `type` check?
I think it doesn't matter if the threshold is smart formatted. Can we verify pls?

Reply:
@balajimanoharan threshold will look weird with smartformat